draft-ietf-acme-acme.md

```diff
@@ -2597,6 +2618,33 @@ within their trusted network and use these resolvers both for both CAA record
 lookups and all record lookups in furtherance of a challenge scheme (A, AAAA,
 TXT, etc.).
 
+## External Accounts
+
+ACME accounts can be associated with accounts in other systems using the
+"external-account-token" field in account objects.  This field can hold any
+value defined by the CA.  The security implications of the field depend on how
+the CA structures its tokens.
+
+If the CA simply provides the client with a random value to use in this field,
+then there is a risk that the value will be stolen and misused.  For example, if
+the CA uses a CDN to terminate HTTPS for ACME, then the CDN could steal a token
+and use it to bind real customer's account to the CDN's ACME account (instead of
+the real customer's ACME account).  CAs that construct tokens in this way SHOULD
+avoid the use of intermediaries that terminate HTTPS.
+
+CAs can mitigate this risk by reducing the privilege associated with the token.
+Instead of granting the holder of the token the full privileges associated with
+the account, for example, the CA could simply use the association for
+record-keeping, so that the external account holder would be notified of actions
+by the ACME client, and the ACME client would still have to acquire privileges
+on its own (proving control of identifiers, etc.).
+
+CAs SHOULD construct tokens in a way that binds them to a particular ACME
+account key pair.  For example, a CA might use JWTs containing a thumbprint of
+the account public key {{?RFC7519}}.  These tokens can safely be used even in
+the presence of HTTPS-terminating intermediaries, since they can only be used by
+the holder of the corresponding account private key.
```

Comment:
Note that this design is only safe because the request bearing the external account token is covered by a signature.